mrun

```diff
@@ -1,3 +1,2 @@
 #! /bin/sh
-
-exec python "mesos_scheduler.py" --mpi "$@"
+exec python "$PARACEL_INSTALL_PREFIX/mesos_scheduler.py" --mpi "$@"
```

Comment:
but `PARACEL_INSTALL_PREFIX` only available during `make`.
I think just `$(dirname "$0")` is enough